hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationRegistry.java

```diff
@@ -89,18 +91,17 @@ long getLastCallId() {
      * Registers an invocation.
      *
      * @param invocation The invocation to register.
+     * @return true if InvocationRegistry is alive when invocation is registered, false otherwise
```

Comment:
you're right, it creates a perception like invocation is not registered when false is returned. 
Maybe we can throw exception instead of returning boolean.